en/studio/quick-starts/single-layer-web-application.md

```diff
@@ -0,0 +1,123 @@
+
+We will use the *Application (Single Layer)* solution template for this tutorial, so pick it and click the *Next* button:
+
+![abp-studio-new-solution-dialog-solution-properties](images/abp-studio-no-layers-new-solution-dialog-solution-properties.png)
+
+In that screen, you choose a name for your solution. You can use different level of namespaces; e.g. `BookStore`, `Acme.BookStore` or `Acme.Retail.BookStore`.
+
+Then select an *output folder* to create your solution. The *Create solution folder* option will create a folder in the given output folder with the same name of your solution.
+
+Once your configuration is done, click the *Next* button to navigate to the *UI Framework* selection:
+
+![abp-studio-new-solution-dialog-ui-framework](images/abp-studio-no-layers-new-solution-dialog-ui-framework.png)
+
+Here, you see all the possible UI options supported by that startup solution template. Pick the **{{ UI_Value }}**.
+
+Notice that; Once you select a UI type, some additional options will be available under the UI Framework list. You can further configure the options or leave them as default and click the *Next* button for the *UI Theme* selection screen:
+
+![abp-studio-new-solution-dialog-ui-theme](images/abp-studio-nolayers-new-solution-dialog-ui-theme.png)
+
+LeptonX is the suggested UI theme that is proper for production usage. Select one of the theme and configure additional options and click the *Next* button for the *Database Provider* selection:
+
+{{ if DB == "EF" }}
+![abp-studio-new-solution-dialog-database-provider](images/abp-studio-no-layers-new-solution-dialog-database-provider-efcore.png)
+{{ else }}
+![abp-studio-new-solution-dialog-database-provider](images/abp-studio-no-layers-new-solution-dialog-database-provider-mongo.png)
+{{ end }}
+
+On that screen, you can decide on your database provider by selecting one of the provided options. There are some additional options for each database provider. Leave them as default or change them based on your preferences, then click the *Next* button for additional *Database Configurations*:
+
+{{ if DB == "EF" }}
+![abp-studio-new-solution-dialog-database-configurations](images/abp-studio-no-layers-new-solution-dialog-database-configurations-efcore.png)
+{{ else }}
+![abp-studio-new-solution-dialog-database-configurations](images/abp-studio-no-layers-new-solution-dialog-database-configurations-mongo.png)
+{{ end }}
+
+Here, you can select the DBMS (only for Entity Framework Core) and the connection string. Now, we are ready to allow ABP Studio to create our solution. Just click the *Create* button and let it go.
+
+After clicking the Create button, the dialog is closed and your solution is loaded into ABP Studio:
+
+![abp-studio-created-new-solution](images/abp-studio-no-layers-created-new-solution.png)
+
+You can explore the solution, but you need to wait for background tasks to be completed before running any application in the solution.
+
```

Comment:
Do you think there is a need to add a title "Exploring the Solution" here like in the [microservice document](https://github.com/abpio/abp-commercial-docs/blob/dev/en/studio/quick-starts/microservice.md#exploring-the-solution)?

Reply:
No, because it's a single-layer-app document. There is only one project in the solution, and we explain it in the [Running the Application](https://github.com/abpio/abp-commercial-docs/blob/5b7388984e34e18f1381b1d21d798a77e6337048/en/studio/quick-starts/single-layer-web-application.md#running-the-application) section. The [layered document](https://github.com/abpio/abp-commercial-docs/blob/5b7388984e34e18f1381b1d21d798a77e6337048/en/studio/quick-starts/layered-web-application.md#running-the-application) does the same thing as well.